### PR TITLE
Update token reference to use labels

### LINKED ISSUE #990: Update token docs
Update the tokens reference so that the custom/extended attributes in the examples use annotations

### PR DESCRIPTION
**Pull request by apaskulin**
## Description

Updates the token reference to use the correct syntax and process for labels instead of custom attributes

## Motivation and Context

Closes #990

### CODE REVIEW

File: content/sensu-go/5.0/reference/tokens.md [hunk 1]
```diff
@@ -16,32 +16,32 @@ menu:
...
 
-Tokens are invoked by wrapping references to entity or custom attributes with double curly braces, such as `{{ .ID }}` to substitute an entity's ID value. Nested Sensu [entity attributes][3] can be accessed via dot notation (ex: `System.Arch`).
+Tokens are invoked by wrapping references to entity attributes and labels with double curly braces, such as `{{ .name }}` to substitute an entity's name. Nested Sensu [entity attributes][3] can be accessed via dot notation (ex: `system.arch`).
 
-- `{{ .ID }}` would be replaced with the [entity `ID` attribute][3]
-- `{{ .URL }}` would be replaced with a custom attribute called `url`
-- `{{ .Disk.Warning }}` would be replaced with a custom attribute called
+- `{{ .name }}` would be replaced with the [entity `name` attribute][3]
+- `{{ .url }}` would be replaced with a custom label called `url`
+- `{{ .disk.warning }}` would be replaced with a custom label called
```
Comment (apaskulin) [inline content/sensu-go/5.0/reference/tokens.md]: It was my understanding that labels are required to be a flat map of key value pairs and that annotations cannot be accessed by tokens. How should we correct this example?

Comment (echlebek) [reply content/sensu-go/5.0/reference/tokens.md]: `{{ .labels.disk-warning }}` IMO

File: content/sensu-go/5.0/reference/tokens.md [hunk 2]
```diff
@@ -94,7 +94,12 @@ tokens declared above.
   "metadata": {
     "name": "example-hostname",
     "namespace": "staging",
-    "labels": null,
+    "labels": {
+      "disk": {
```
Comment (echlebek) [inline content/sensu-go/5.0/reference/tokens.md]: This should be

```
"labels": {
  "disk-warning": "80",
  "disk-critical": "90"
}
```

File: content/sensu-go/5.0/reference/tokens.md (hunk 1 above)
Comment (apaskulin) [reply content/sensu-go/5.0/reference/tokens.md]: Perfect. Thanks!

**Comment by apaskulin** [comment]
@echlebek Thanks for your review! Updated this to use the `.labels.thing-thing` syntax.

File: content/sensu-go/5.0/reference/tokens.md [hunk 3]
```diff
@@ -51,20 +51,20 @@ Check config token errors will be logged by the agent, and sent to Sensu backend
...
   "metadata": {
     "name": "check-disk-usage",
-    "namespace": "{{ .Namespace | default \"production\" }}",
+    "namespace": "{{ .namespace | default \"production\" }}",
     "labels": null,
     "annotations": null
   },
   "spec": {
-    "command": "check-disk-usage.rb -w {{.Disk.Warning | default 80}} -c {{.Disk.Critical | default 90}}",
+    "command": "check-disk-usage.rb -w {{.disk.warning | default 80}} -c {{.disk.critical | default 90}}",
```
Comment (echlebek) [inline content/sensu-go/5.0/reference/tokens.md]: This one should be `.labels.disk-warning` and `.labels.disk-critical` too, I think.

**Review by echlebek — APPROVED** [review]
Looks good, one minor change needed and then :ship:

**Pull request merged**